01-node-tutorial/answers/01-intro.js

```diff
@@ -0,0 +1,5 @@
+const txt = "Hello World!";
+
+if (txt.length > 0) {
+    console.log(txt.toUpperCase());
+}
```

Comment:
You may be aware of this but all of these red circles just mean git likes there to be an empty line at the end of the file.

Reply:
I didn't pay attention to this before, but it really will make the file more readable. Thanks